.gitignore

```diff
@@ -40,3 +40,5 @@ TestResult.xml
 *.err
 GlobalAssemblyInfo.cs
 CommonAssemblyInfo.cs
+All Tests in CorrugatedIron.dcvr.settings
```

Comment:
Hmm, that was generated from the GitHub app, and it does ignore the file. (shrug) 
Will have to see how the filename glob matching works in windows.